### PR TITLE
IGNITE-12519: Add support for lowercase object names

### PR DESCRIPTION
**Pull request by Flaugh24**

### CODE REVIEW

**Review by aealeksandrov — APPROVED** [review]
LGTM

File: modules/spark-3.2-ext/spark-3.2/src/main/scala/org/apache/ignite/spark/impl/QueryUtils.scala [hunk 1]
```diff
@@ -26,6 +26,17 @@ import org.apache.spark.sql.types._
   * Utility class for building SQL queries.
   */
 private[impl] object QueryUtils extends Logging {
+    /**
+      * @param str String.
```
Comment (sk0x50) [inline modules/spark-3.2-ext/spark-3.2/src/main/scala/org/apache/ignite/spark/impl/QueryUtils.scala:30]: This scaladoc does not provide any useful information. Please update.

File: modules/spark-3.2-ext/spark-3.2/src/main/scala/org/apache/ignite/spark/impl/optimization/accumulator/package.scala [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package org.apache.ignite.spark.impl.optimization
```
Comment (sk0x50) [inline modules/spark-3.2-ext/spark-3.2/src/main/scala/org/apache/ignite/spark/impl/optimization/accumulator/package.scala:1]: Please add license header.

File: modules/spark-3.2-ext/spark-3.2/src/test/scala/org/apache/ignite/spark/IgniteQueryCompilatorSpec.scala [hunk 3]
```diff
@@ -0,0 +1,122 @@
+package org.apache.ignite.spark
```
Comment (sk0x50) [inline modules/spark-3.2-ext/spark-3.2/src/test/scala/org/apache/ignite/spark/IgniteQueryCompilatorSpec.scala:1]: Please add license header

File: modules/spark-3.2-ext/spark-3.2/src/main/scala/org/apache/ignite/spark/impl/QueryUtils.scala (hunk 1 above)
Comment (Flaugh24) [reply modules/spark-3.2-ext/spark-3.2/src/main/scala/org/apache/ignite/spark/impl/QueryUtils.scala:30]: done

File: modules/spark-3.2-ext/spark-3.2/src/main/scala/org/apache/ignite/spark/impl/optimization/accumulator/package.scala (hunk 2 above)
Comment (Flaugh24) [reply modules/spark-3.2-ext/spark-3.2/src/main/scala/org/apache/ignite/spark/impl/optimization/accumulator/package.scala:18]: done

**Pull request closed without merging**